portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -108,7 +108,7 @@ private  void createList(PreparedQuery results) {
   }
 
   // Adds a comment into the Servlet's internal data.
-  private void addComment(Comment comment) {
+  public void addComment(Comment comment) {
```

Comment:
Alternatively you could make these package private (e.g. `void addComment(Comment comment)`) and then move the test to `src/test/java/com/google/sps/servlets/PortfolioTest.java` and change the package to `com.google.sps.servlets`. This way you can restrict the visibility of the methods to the test. You can do something similar for `convertToJson()`.